Documentation/policy/language.rst

```diff
@@ -845,6 +850,12 @@ latter rule will have no effect.
 .. note:: Layer 7 rules are not currently supported in `HostPolicies`, i.e.,
           policies that use :ref:`NodeSelector`.
 
+.. note:: Layer 7 policies --and pod annotations-- result in traffic being
+   proxified through an Envoy instance provided in each Cilium agent pod.
```

Comment:
Proxied would be the usual term for this. I saw before that the dictionary didn't include that term, but it is a more natural one than proxified.
```suggestion
   proxied through an Envoy instance provided in each Cilium agent pod.
```